Doesn't run with latest Node version(s)

I did this:
```npm i -g mongo-express@git+https://github.com/mongo-express/mongo-express.git#master```

install succeeded, then I did this:
```mongo-express --version```

and this is the response:
```
node:fs:603
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open './build-assets.json'
    at Object.openSync (node:fs:603:3)
    at Object.readFileSync (node:fs:471:35)
    at file:///usr/local/share/.config/yarn/global/node_modules/mongo-express/lib/middleware.js:9:30
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './build-assets.json'
}
```

This project was the best thing I found (and used for years) for managing / exploring a MongoDB instance. Why is it so broken now?

I tried this again today, uninstalled mongo-express globally and cloned the master branch for a clean start.

```
root@XXXXXXX:~/mongo-express# npm run build

> mongo-express@1.0.0 build
> NODE_ENV=production webpack

CLI for webpack must be installed.
  webpack-cli (https://github.com/webpack/webpack-cli)

We will use "yarn" to install the CLI via "yarn add -D webpack-cli".
Do you want to install 'webpack-cli' (yes/no): yes
Installing 'webpack-cli' (running 'yarn add -D webpack-cli')...
/bin/sh: 1: yarn: not found
undefined
root@XXXXXXX:~/mongo-express# npm i -g yarn

added 1 package in 2s
root@XXXXXXX:~/mongo-express# npm run build

> mongo-express@1.0.0 build
> NODE_ENV=production webpack

CLI for webpack must be installed.
  webpack-cli (https://github.com/webpack/webpack-cli)

We will use "yarn" to install the CLI via "yarn add -D webpack-cli".
Do you want to install 'webpack-cli' (yes/no): yes
Installing 'webpack-cli' (running 'yarn add -D webpack-cli')...
➤ YN0000: ┌ Resolution step
➤ YN0060: │ mongo-express@workspace:. provides babel-loader (paccad) with version 9.1.3, which doesn't satisfy what @cypress/code-coverage and some of its descendants request
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 0s 741ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ cli-truncate@npm:2.1.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ cliui@npm:6.0.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ cliui@npm:7.0.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ cliui@npm:8.0.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ clone-deep@npm:4.0.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: ⠦ ==============================--------------------------------------------------
Error: Cannot find module 'webpack-cli/package.json'
Require stack:
- /root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Function.resolve (node:internal/modules/cjs/helpers:127:19)
    at runCli (/root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js:65:26)
    at /root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js:154:5
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js'
  ]
}
```

> I tried this again today, uninstalled mongo-express globally and cloned the master branch for a clean start. Tried doing `npm run build` and it made me install yarn. Output is below culminating in `Cannot find module 'webpack-cli/package.json'`.
> 
> ```
> root@XXXXXXX:~/mongo-express# npm run build
> 
> > mongo-express@1.0.0 build
> > NODE_ENV=production webpack
> 
> CLI for webpack must be installed.
>   webpack-cli (https://github.com/webpack/webpack-cli)
> 
> We will use "yarn" to install the CLI via "yarn add -D webpack-cli".
> Do you want to install 'webpack-cli' (yes/no): yes
> Installing 'webpack-cli' (running 'yarn add -D webpack-cli')...
> /bin/sh: 1: yarn: not found
> undefined
> root@XXXXXXX:~/mongo-express# npm i -g yarn
> 
> added 1 package in 2s
> root@XXXXXXX:~/mongo-express# npm run build
> 
> > mongo-express@1.0.0 build
> > NODE_ENV=production webpack
> 
> CLI for webpack must be installed.
>   webpack-cli (https://github.com/webpack/webpack-cli)
> 
> We will use "yarn" to install the CLI via "yarn add -D webpack-cli".
> Do you want to install 'webpack-cli' (yes/no): yes
> Installing 'webpack-cli' (running 'yarn add -D webpack-cli')...
> ➤ YN0000: ┌ Resolution step
> ➤ YN0060: │ mongo-express@workspace:. provides babel-loader (paccad) with version 9.1.3, which doesn't satisfy what @cypress/code-coverage and some of its descendants request
> ➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
> ➤ YN0000: └ Completed in 0s 741ms
> ➤ YN0000: ┌ Fetch step
> ➤ YN0013: │ cli-truncate@npm:2.1.0 can't be found in the cache and will be fetched from the remote registry
> ➤ YN0013: │ cliui@npm:6.0.0 can't be found in the cache and will be fetched from the remote registry
> ➤ YN0013: │ cliui@npm:7.0.4 can't be found in the cache and will be fetched from the remote registry
> ➤ YN0013: │ cliui@npm:8.0.1 can't be found in the cache and will be fetched from the remote registry
> ➤ YN0013: │ clone-deep@npm:4.0.1 can't be found in the cache and will be fetched from the remote registry
> ➤ YN0000: ⠦ ==============================--------------------------------------------------
> Error: Cannot find module 'webpack-cli/package.json'
> Require stack:
> - /root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js
>     at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
>     at Function.resolve (node:internal/modules/cjs/helpers:127:19)
>     at runCli (/root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js:65:26)
>     at /root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js:154:5
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
>   code: 'MODULE_NOT_FOUND',
>   requireStack: [
>     '/root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js'
>   ]
> }
> ```

Run first `npm install` then `npm run build`.

@rtritto I tried from a clean checkout of the repo again:

```
root@XXXXX:~/mongo-express# npm install
Killed####⠂⠂⠂⠂⠂⠂⠂⠂⠂) ⠦ reify:fsevents: sill reify mark deleted [ '/root/mongo-express/node_modules/fsevents' ]
root@XXXXX:~/mongo-express# 
root@XXXXX:~/mongo-express# 
root@XXXXX:~/mongo-express# npm run build

> mongo-express@1.0.0 build
> NODE_ENV=production webpack

CLI for webpack must be installed.
  webpack-cli (https://github.com/webpack/webpack-cli)

We will use "npm" to install the CLI via "npm install -D webpack-cli".
Do you want to install 'webpack-cli' (yes/no): yes
Installing 'webpack-cli' (running 'npm install -D webpack-cli')...
npm WARN deprecated csurf@1.11.0: Please use another csrf package

added 269 packages, removed 631 packages, changed 483 packages, and audited 431 packages in 1m

28 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Error: Cannot find module 'webpack-cli/package.json'
Require stack:
- /root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Function.resolve (node:internal/modules/cjs/helpers:127:19)
    at runCli (/root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js:65:26)
    at /root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js:154:5
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/root/.nvm/versions/node/v18.17.1/lib/node_modules/webpack/bin/webpack.js'
  ]
}
root@XXXXX:~/mongo-express# 
```

--- a/lib/middleware.js
+++ b/lib/middleware.js
@@ -6,12 +6,10 @@ import fileUpload from 'express-fileupload';
 import * as swigFilters from './filters.js';
 import router from './router.js';
 
-const assets = JSON.parse(fs.readFileSync('./build-assets.json'));
-
 const middleware = async function (config) {
   const app = express();
 
-  app.locals.assets = assets;
+  app.locals.assets = JSON.parse(fs.readFileSync('./build-assets.json'));
 
   // Set up swig
   const swigOptions = {

